apps/likes/laboite/apps/likes/models.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+import requests
+
+from boites.models import App
+from . import settings
+
+
+class AppLikes(App):
+    UPDATE_INTERVAL = 1
```

Comment:
UPDATE_INTERVAL is by default in seconds, is it what you want? Or are you missing the unit like `1 * MINUTE`?